tc/app/aws/get_ecr_images_list

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# shellcheck source=./tc/framework/functions
+# shellcheck source=./tc/app/aws/functions
+# shellcheck disable=SC1091,SC2034
+
+source "$CLI_DIR/tc/framework/functions"
+source "$CLI_DIR/tc/app/aws/functions"
+
+set +e
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --profile)
+    PROFILE="$2"
+    shift
+    shift
+    ;;
+    --repository)
+    REPOSITORY="$2"
+    shift
+    shift
+    ;;
+    *)
+    ;;
+esac
+done
+
+function _get_ecr_images_list() {
+  __output=$( _describe_ecr_images \
+  | jq -r '.imageDetails[] | (.imagePushedAt|tostring) + " " + .imageDigest + " " + .imageTags[0]' \
+  | sort | awk '{print $3}' )
+  export _error_code=$?
+  __MSG="Command: \033[36mtcctl aws ecr list-images\033[39m\n"
+  __MSG+="$__output"
+  if [[ $_error_code -eq "0" ]]; then
```

Comment:
but no quotes here.
`-eq` is for numeric comparison use `=` for strings. [[ ]]` doesn't require quoting at all. ('==' works only for bash btw)
Please pay attention to general code style.